packages/pages/src/common/src/project/structure.ts

```diff
@@ -128,7 +125,7 @@ export class ProjectStructure {
     this.sitesConfigRoot = new Path(
       this.#config.filepathsConfig.sitesConfigRoot
     );
-    this.templatesRoot = new Path(this.#config.filepathsConfig.templatesRoot);
+    this.templatesRoot = new Path("src/templates");
```

Comment:
is there a single file (like a constants file or something) where we can define `src/templates` as the root path for templates so it can be referred to everywhere? that way, if we made an update to the path, we wouldn't need to remember to update each place individually

Reply:
The `ProjectStructure` type is used as the config as well as internally in some of the functions so to remove it for the config meant to hardcode it in the function. I considered including it in the type as 
```
interface ProjectStructure {
  templatesRoot: 'src/templates'
  somthingElse: string
  ...
}
```
but after seeing there are several type declarations for `ProjectStructure`, one of which where `templatesRoot` is of type `Path` and each hardcoding is in its own sub-repo(ish) directory, I decided against it.

What are your thoughts regarding this?

Reply:
actually, it looks like there is a `pages/packages/pages/src/dev/server/ssr/constants.ts` file. it already has the template path hardcoded. what do you think of moving that file a few folders up so it can be shared, or making a similar constants file in `pages/packages/pages` or `pages/packages/pages/src` with the template path?

Reply:
I made a new constants file

Reply:
outdated